src/box/lua/config/descriptions.lua

```diff
@@ -0,0 +1,2446 @@
+    Defines the Lua functions that the user has permission to call. This
+    field accepts a special value, `all`, which grants the privilege
+    to use any global non-built-in Lua functions.
+]])
+M['credentials.users.*.privileges.*.lua_call.*'] = 'Lua function name.'
+
+M['credentials.users.*.privileges.*.lua_eval'] = format_text([[
+    Whether this user can execute arbitrary Lua code.
+]])
+
+M['credentials.users.*.privileges.*.permissions'] = format_text([[
+    Permissions assigned to this user or a user with this role.
+]])
+
+M['credentials.users.*.privileges.*.permissions.*'] = 'Permission name.'
+
+M['credentials.users.*.privileges.*.sequences'] = format_text([[
+    Sequences to which this user gets the specified permissions.
+]])
+
+M['credentials.users.*.privileges.*.sequences.*'] = 'Sequence name.'
+
+M['credentials.users.*.privileges.*.spaces'] = format_text([[
+    Spaces to which this user gets the specified permissions.
+]])
+
+M['credentials.users.*.privileges.*.spaces.*'] = 'Space name.'
+
+M['credentials.users.*.privileges.*.sql'] = format_text([[
+    Whether this user can execute an arbitrary SQL expression.
+]])
+
+M['credentials.users.*.privileges.*.sql.*'] = 'SQL expression name.'
+
+M['credentials.users.*.privileges.*.universe'] =
+    M['credentials.roles.*.privileges.*.universe']
+
+M['credentials.users.*.roles'] = 'An array of roles granted to this user.'
+
+M['credentials.users.*.roles.*'] = 'Role name.'
+
+-- }}} credentials.users configuration
+
+-- }}} credentials configuration
+
+-- {{{ database configuration
+
+M['database'] = format_text([[
+    The `database` section defines database-specific configuration parameters,
+    such as an instance's read-write mode or transaction isolation level.
+]])
+
+M['database.hot_standby'] = format_text([[
+    Whether to start the server in the hot standby mode. This mode can be used
+    to provide failover without replication.
+
+    Note: `database.hot_standby` has no effect:
+
+    - If `wal.mode` is set to none.
+    - If `wal.dir_rescan_delay` is set to a large value on macOS or FreeBSD.
+    On these platforms, the hot standby mode is designed so that the loop
+    repeats every `wal.dir_rescan_delay` seconds.
+    - For spaces created with engine set to `vinyl`.
+]])
+
+M['database.instance_uuid'] = format_text([[
+    An instance UUID.
+
+    By default, instance UUIDs are generated automatically.
+    `database.instance_uuid` can be used to specify an instance identifier
+    manually.
+
+    UUIDs should follow these rules:
+
+    - The values must be true unique identifiers, not shared by other instances
+    or replica sets within the common infrastructure.
+    - The values must be used consistently, not changed after the initial setup.
+    The initial values are stored in snapshot files and are checked whenever
+    the system is restarted.
+    - The values must comply with RFC 4122. The nil UUID is not allowed.
+]])
```

Comment:
It worth to note that upgrading from 2.x requires to set instance and replicaset UUIDs until the database schema is upgraded. After the full upgrade, the UUIDs can be removed from the configuration.